src/block.ts

```diff
@@ -0,0 +1,211 @@
+import { bytes as binary, CID } from './index.js'
```

Comment:
it's a shame GH hasn't picked up the rename here, makes it hard to review without seeing the diff like most of the other files